fastlane/lib/fastlane/actions/upload_to_testflight.rb

```diff
@@ -15,10 +15,6 @@ def self.run(values)
           values[:ipa] = File.expand_path(values[:ipa]) if values[:ipa]
         end
 
-        if values[:api_key_path].nil?
-          values[:api_key] ||= Actions.lane_context[SharedValues::APP_STORE_CONNECT_API_KEY]
-        end
```

Comment:
Not needed anymore because `Pilot` will look for existing `Spaceship::ConnectAPI.client` now and use that